HW27/document.js

```diff
@@ -0,0 +1,62 @@
+const myDiv = document.createElement('div');
+myDiv.className = 'buttons';
+myDiv.style.background = 'white';
+
+['Add to friends', 'Write message', 'Job offer', 'Add HW'].map(buttonName => {
+    let button = document.createElement('button');
+    button.innerText = `${buttonName}`;
+    button.style.cssText = 'margin-left: 65px; margin-bottom: 5px; border-radius: 10px; background-color: LightGrey;';
+    myDiv.appendChild(button);
+})
+
+document.getElementsByTagName('body')[0].appendChild(myDiv);
+
+const btn2 = document.getElementsByTagName('button')[1];
+btn2.style.backgroundColor = 'Aquamarine';
+
+let counter = Math.floor(Math.random() * 150);
+
+const myAmnt = document.createElement('div');
+myAmnt.className = 'counter';
+myAmnt.style.background = 'white';
+myAmnt.innerText = `Friends: ${counter}`;
+
+document.getElementsByTagName('body')[0].appendChild(myAmnt);
+
+const btn = document.getElementsByTagName('button')[0];
+
+
+btn.addEventListener("click", (Btnevent) => {
+    counter += 1;
+    myAmnt.innerText = `Friends: ${counter}`
+    btn.innerText = 'Waiting for confirmation';
+    Btnevent.target.disabled = true;
+    });
+
+btn2.addEventListener("click", (Btnevent) => {
+    btn2.style.backgroundColor = 'LightGrey';
+})
+
+const btn3 = document.getElementsByTagName('button')[2];
+btn3.addEventListener("click", (Btnevent) => {
+    btn.style.display = ((btn.style.display!='none') ? 'none' : 'block');
```

Comment:
After a second click here button appears again, but not in the initial position. To fix this, you can use `inline` instead of `block` for `display` value